<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogJump</class>
 <widget class="QDialog" name="DialogJump">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxFinalPositionOffset">
         <property name="title">
          <string>Final position offset</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1">
            <item row="0" column="0">
             <widget class="QLabel" name="labelX">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="PanelPrimitiveValue" name="panelPrimitiveX" native="true">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="4">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>128</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelY">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="PanelPrimitiveValue" name="panelPrimitiveY" native="true">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelYPlus">
              <property name="text">
               <string>Y plus:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="PanelPrimitiveValue" name="panelPrimitiveYPlus" native="true">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelZ">
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="PanelPrimitiveValue" name="panelPrimitiveZ" native="true">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxPeak">
         <property name="title">
          <string>Peak</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="labelPeakY">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
              <item>
               <widget class="PanelPrimitiveValue" name="panelPrimitivePeakY" native="true">
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>28</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelPeakYPlus">
              <property name="text">
               <string>Y plus:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
              <item>
               <widget class="PanelPrimitiveValue" name="panelPrimitivePeakYPlus" native="true">
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogJump</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogJump</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
